frontend/test/__support__/e2e/helpers/e2e-notebook-helpers.js

```diff
@@ -32,3 +34,30 @@ export function visualize(callback) {
     }
   });
 }
+
+export function addSummaryField({ metric, field, stage = 0, index = 0 }) {
+  getNotebookStep("summarize", { stage, index })
+    .findByTestId("aggregate-step")
+    .findAllByTestId("notebook-cell-item")
+    .last()
```

Comment:
Would it be possible to explicitly search for a "+" button here instead of relying it's the last one?